pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -1094,7 +1094,7 @@ public void setMaxConsumersPerTopic(@PathParam("property") String property, @Pat
     @ApiOperation(value = "Get maxConsumersPerSubscription config on a namespace.")
     @ApiResponses(value = {@ApiResponse(code = 403, message = "Don't have admin permission"),
             @ApiResponse(code = 404, message = "Namespace does not exist")})
-    public int getMaxConsumersPerSubscription(@PathParam("property") String property,
+    public Integer getMaxConsumersPerSubscription(@PathParam("property") String property,
```

Comment:
**Note**: 
The `maxConsumersPerSubscription` will become `null` When the `removeMaxConsumersPerSubscription` method is executed, and then `NPE` will appear when get maxConsumersPerSubscription, so we need to change `int` to `Integer`.